AutomatedTesting/Gem/PythonTests/scripting/EditMenu_Default_UndoRedo.py

```diff
@@ -49,58 +35,39 @@ class EditMenu_Default_UndoRedo:
 
     :return: None
     """
-    def __init__(self):
-        self.editor_main_window = None
-        self.sc_editor = None
-        self.sc_editor_main_window = None
-        self.variable_manager = None
-
-    def get_graph_variables_row_count(self, graph_variables):
-        return graph_variables.model().rowCount(QtCore.QModelIndex())
-
-
-    @pyside_utils.wrap_async
-    async def run_test(self):
 
-        # Preconditions
-        general.idle_enable(True)
-        scripting_tools.initialize_editor_object(self)
+    # Preconditions
+    import azlmbr.legacy.general as general
+    import editor_python_test_tools.script_canvas_tools_qt as sc_tools_qt
 
-        # 1) Open Script Canvas window and initialize the qt SC objects
-        general.open_pane(SCRIPT_CANVAS_UI)
-        helper.wait_for_condition(lambda: general.is_pane_visible(SCRIPT_CANVAS_UI), WAIT_TIME_3)
-        scripting_tools.initialize_sc_editor_objects(self)
+    general.idle_enable(True)
 
-        # 2) Open Variable Manager if not opened already
-        scripting_tools.initialize_variable_manager_object(self)
+    # 1) Open Script Canvas window
+    qtpy_o3de_editor = sc_tools_qt.qtpy_o3de_editor
+    qtpy_o3de_editor.open_script_canvas()
 
-        # 3) Create Graph
-        scripting_tools.create_new_sc_graph(self.sc_editor_main_window)
+    # 2) Create Graph
+    sc_editor = qtpy_o3de_editor.sc_editor
```

Comment:
could this be returned by `open_script_canvas()`?

Reply:
updated the function to return a reference to the script canvas editor